src/domain/validatecustomforms/repositories/CustomFormErrorsRepository.ts

```diff
@@ -0,0 +1,3 @@
+export interface CustomFormErrorsRepository {
```

Comment:
[no need to change] That's too specific for a repository. It's unlikely we will be able to add more methods to it. An alternative would would be a repo called `DataSetsRepository` having a `validate(id: Id): string[]` method.